Add some terms to lint file

c0ffeeca7 opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
- Add a few terms to the lint file. Goal: avoid typos in product names or abbreviations in future commits.



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - [x]  I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

c0ffeeca7 committed:
Add some terms to lint file

frenck commented on .textlintrc.json:
```diff
@@ -126,6 +127,8 @@
         "Home Assistant Supervised",
         "Home Assistant Supervisor",
         "Home Assistant",
+        "Home Assistant SkyConnect",
```
Maybe add `SkyConnect` as a single word too?

c0ffeeca7 replied:
Done. Thank you for the hint!

frenck reviewed (changes requested):
Looks like it found issues that need to be addressed as well in that case:

![image](https://user-images.githubusercontent.com/195327/224029574-133ab743-6a0a-4f59-812a-01d82066053d.png)

c0ffeeca7 committed:
Remove Zigbee2MQTT from lint file

- to avoid conflict with currently used variants in path names and version names
- Fix typo in mqtt documentation

c0ffeeca7 committed:
Add SkyConnect to lint file

- fix typo in mqtt documentation

frenck commented:
Interesting, case of zigbee2mqtt, the linter should not have triggered on that.
Seems like it was a known issue before: <https://github.com/sapegin/textlint-rule-terminology/issues/38>

frenck reviewed (approved):
Thanks, @c0ffeeca7 👍 

../Frenck
